specification/marketplace/resource-manager/Microsoft.Marketplace/stable/2020-01-01/Marketplace.json

```diff
@@ -0,0 +1,552 @@
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/OfferListResponse"
+            }
+          },
+          "default": {
+            "description": "Microsoft.Marketplace error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "GetPrivateStoreOffers": {
+            "$ref": "./examples/GetPrivateStoreOffers.json"
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Marketplace/PrivateStores/{PrivateStoreId}/Offers/{OfferId}": {
+      "parameters": [
+        {
+          "$ref": "#/parameters/OfferIdPathParameter"
+        },
+        {
+          "$ref": "#/parameters/PrivateStoreIdParameter"
+        },
+        {
+          "$ref": "#/parameters/ApiVersionParameter"
+        }
+      ],
+      "delete": {
+        "tags": [
+          "PrivateStores"
+        ],
+        "operationId": "PrivateStoreOffer_Delete",
+        "description": "Deletes an offer from the given private store.",
+        "responses": {
+          "204": {
+            "description": "Offer was deleted successfully"
+          },
+          "default": {
+            "description": "Microsoft.Marketplace error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "DeletePrivateStoreOffer": {
+            "$ref": "./examples/DeletePrivateStoreOffer.json"
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "PrivateStores"
+        ],
+        "operationId": "PrivateStoreOffer_Get",
+        "description": "Gets information about a specific offer.",
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "Offer information retrieved",
+            "schema": {
+              "$ref": "#/definitions/Offer"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "GetPrivateStoreOffer": {
+            "$ref": "./examples/GetPrivateStoreOffer.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "PrivateStores"
+        ],
+        "operationId": "PrivateStoreOffer_CreateOrUpdate",
+        "description": "Update or add an offer to the default collection of the private store.",
+        "consumes": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "body",
+            "name": "payload",
+            "schema": {
+              "$ref": "#/definitions/OfferUpdateProperties"
```

Comment:
>OfferUpdateProperties [](start = 37, length = 21)

Also, both PUT and GET must have the same definition that they accept in body (PUT) and in response (PUT and GET)